WinUIGallery/Helper/NavigationHelper.cs

```diff
@@ -168,14 +169,15 @@ public void OnNavigatedFrom(NavigationEventArgs e)
     public class RootFrameNavigationHelper
     {
         private Frame Frame { get; set; }
-        private Microsoft.UI.Xaml.Controls.NavigationView CurrentNavView { get; set; }
+        private NavigationView CurrentNavView { get; set; }
+        private bool alreadyProcessedKeyDown = false;
```

Comment:
nit: this should follow the naming convention `keyDownProcessed` or `isKeyDownProcessed`

Reply:
Re-opening as this hasn't been addressed yet.

Reply:
Oh you are right, I've updated it to `isKeyDownProcessed`.